services/web/server/src/simcore_service_webserver/payments/_rpc.py

```diff
@@ -0,0 +1,80 @@
+""" PRC client-side for the RPC server at the payments service
+
+"""
+
+import logging
+from decimal import Decimal
+
+from aiohttp import web
+from models_library.api_schemas_payments import PAYMENTS_RPC_NAMESPACE
+from models_library.api_schemas_webserver.wallets import WalletPaymentCreated
+from models_library.users import UserID
+from models_library.wallets import WalletID
+from pydantic import parse_obj_as
+from servicelib.logging_utils import log_decorator
+from servicelib.rabbitmq import RabbitMQRPCClient, RPCMethodName
+
+from ..rabbitmq_settings import RabbitSettings
+from ..rabbitmq_settings import get_plugin_settings as get_rabbitmq_settings
+
+_logger = logging.getLogger(__name__)
+
+
+_APP_PAYMENTS_RPC_CLIENT_KEY = f"{__name__}.RabbitMQRPCClient"
+
+
+async def rabbitmq_rpc_client_lifespan(app: web.Application):
+    settings: RabbitSettings = get_rabbitmq_settings(app)
+    rpc_client = await RabbitMQRPCClient.create(
+        client_name="webserver_payments_client",
+        settings=settings,
+    )
+
+    assert rpc_client  # nosec
+    assert rpc_client.client_name == "webserver_payments_client"  # nosec
```

Comment:
are these assertions bringing something here? beside the first one?

Reply:
not really. I took them from a test and did not think about it carefully